index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
```

Comment:
- [ ] It would be a lot more conventional to add a descriptive title that resonates with your page's content to your app. This boosts your SEO ranking and improves your accessibility scores. 🤗

Reply:
Great work on making a descriptive title :blush: